dependencies.xml

```diff
@@ -40,7 +40,7 @@ Note all install methods after "main" take
     <matplotlib>3.2</matplotlib>
     <statsmodels/>
     <cloudpickle>1.6</cloudpickle>
-    <tensorflow>1.15</tensorflow>
+    <tensorflow>2.0</tensorflow>
```

Comment:
Can we loose the constraint on the version?

Reply:
I'll try that.

Reply:
(With the warning that unless we switch to default to the conda-forge channel it won't make much difference (except for pip installs) since conda's regular channel doesn't have anything newer than 2.0)

Reply:
Hm, tensorflow 2.3 gives different classification results resulting in a diff error.  https://civet.inl.gov/job/768524/

Reply:
tensorflow 2.1 also gives different values.  https://civet.inl.gov/job/768517/
It looks like tensorflow 2.1 and 2.3 are the same.
So once we can get everything to something newer than 2.0 we should be fine, but for now I think I will revert to 2.0 since most of our conda machines are stuck at that.

Reply:
That is good. Thanks.